dotnet/src/Connectors/Connectors.OpenAI/AzureSdk/AzureOpenAITextToAudioClient.cs

```diff
@@ -79,7 +79,7 @@ internal sealed class AzureOpenAITextToAudioClient
 
         var binaryData = await BinaryData.FromStreamAsync(stream, cancellationToken).ConfigureAwait(false);
 
-        return new List<AudioContent> { new(binaryData, modelId) };
+        return new List<AudioContent> { new(binaryData.ToMemory(), modelId) };
```

Comment:
nit: There is small unnecessary conversion here, first we read HTTP response as `Stream`, then we convert it to `BinaryData` and then to `ReadOnlyMemory`. 

Instead, we should be able to read HTTP response as `byte[]` and convert it directly to `ReadOnlyMemory`:
```csharp
var bytes = await response.Content.ReadAsByteArrayAndTranslateExceptionAsync().ConfigureAwait(false);

return new List<AudioContent> { new(new(bytes), modelId) };
```

Reply:
fixed